framework/base/Application.php

```diff
@@ -246,6 +246,18 @@ public function preInit(&$config)
             $this->setTimeZone('UTC');
         }
 
+        if (isset($config['container'])) {
```

Comment:
Looks inconsistent: it would be better to use single setter name.
With such structure confuguration for DI definitions may come from 2 separated config keys:
1) container -> definitions
2) containerDefinitions